### DISCUSSION TITLE
telescope.builtin.commands()'s behavior is weird.

### DISCUSSION THREAD

**Discussion by JoeMob** [ayamir/nvimdots#654] [Q&A] [answered]
It's showing all `<A->` in `<M->`, and `<S-.>` in a capitalized `<.>`.
How can I configure telescope to show original key strokes?
I'm confussed since `:help telescope` didn't provide enough information.🤨
![image](https://user-images.githubusercontent.com/49267059/231644381-85a12cf1-851d-4b27-a5b9-9bf7b6282004.png)

**Comment by charliie-dev** [comment, marked as answer]
`:help meta`

![圖片](https://user-images.githubusercontent.com/32497323/231650636-3893aad4-3361-4031-999b-b302c584f64e.png)

**Reply by charliie-dev** [reply]
it's built-in in neovim, telescope has nothing to do with it.
